lib/religion/religion.ts

```diff
@@ -5,12 +5,19 @@ import { AlignmentsAbbreviated, ClericDomains, WorldType } from '../src/worldTyp
 import { RaceName, GenderName, AgeName, NPC, ThresholdTable, PartialRecord, Virtues } from '../'
 
 interface Followers {
-  /** @example 'Zeus is followed by many, of all different race and creed.' */
-  description: string
+  /**
+   * @example 'Zeus is followed by many, of all different race and creed.' 
+   */
+  description: string | string[]
```

Comment:
Does this need to allow string arrays? I couldn't spot any at a glance.

Reply:
No, and I think it'd be good to keep it as a single string rather than an array. I can't think of any instances where we would need to use an array over a string, outside of alternate descriptions.

Reply:
I was mainly thinking of Aphrodite Areia until I realized that Avatars could serve the same Purpose and I forgot to change it back